string filepath

bool domain
bool problem
---
string msg
bool success
